Installing-Oppia-(Linux;-Python-3).md

```diff
@@ -33,7 +34,23 @@ bash scripts/install_prerequisites.sh
 
 2. Verify that Java version 11+ is installed. You can use the command `java -version` to do this. (If you need to change the default binary for the `java` command on your machine, use `sudo update-alternatives --config java`.)
 
-3. Install Chrome from [Google's website](https://www.google.com/chrome). You'll need this to run tests.
+3. Install GraphicsMagick to enable local image compression and ensure tests run successfully. Follow the appropriate steps below based on your operating system:
+   - For Ubuntu/Debian systems:
+      ```
+      sudo apt-get update
+      sudo apt-get install graphicsmagick
+      ```
+   - For MacOS:
+      ```
+      brew install graphicsmagick
+      ```
```

Comment:
The macOS installation instructions should go in our separate macOS installation guide